runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -93,6 +95,14 @@
 	private static final Pattern CLUE_SCROLL_PATTERN = Pattern.compile("You have completed [0-9]+ ([a-z]+) Treasure Trails.");
 	private static final int THEATRE_OF_BLOOD_REGION = 12867;
 
+	// Brimstone loot handling
+	private static final String BRIMSTONE_CHEST_MESSAGE = "You find some treasure in the chest!";
+	private static final String BRIMSTONE_CHEST_EVENT_TYPE = "Brimstone Chest";
+	private static final Pattern VALUABLE_DROP_PATTERN = Pattern.compile("Valuable drop: ([0-9,]+) x ([-A-z-0-9 ]+)");
```

Comment:
Both of these can be one pattern: `Valuable drop: (?:([0-9,]+) x )?([-A-z-0-9 ]+)`